contracts/lib/VerifierMTP.sol

```diff
@@ -22,71 +22,114 @@ pragma solidity >=0.7.0 <0.9.0;
 
 contract VerifierMTP {
     // Scalar field size
-    uint256 constant r    = 21888242871839275222246405745257275088548364400416034343698204186575808495617;
+    uint256 constant r =
```

Comment:
We have this files in .solhintignore, I can just undo this changes. @vmidyllic suggests to additionally run linter on these files.

Reply:
We already have these files in .solhintignore. Formatting was done by `npm run prettier` -> added these files to .prettierignore (since there is no options to disable formatting for block, other option to add `// prettier-ignore` to each variable)